File: dynarr.ts
const prog: number = 100;
const mnoznik: number = 3;

const tab: number[] = [10, 20, 30, 40, 50];
var res: number = 
    tab.map(function (it) {
            return it * mnoznik;
        }
    ).filter(function (it) {
        return it > prog;
    }).reduce(function(wynik, it) {
            return wynik + it;
        },
        0
    );
// res = 270

console.log(res);

File: dynarr.kt
val mnoznik = 3
val prog = 100

fun main() {
    val tab = arrayOf(10, 20, 30, 40, 50)
    val res = tab
        .map { it * mnoznik }
        .filter { it > prog }
        .reduce { acc, it -> acc + it }
    // res = 270
    println(res)
}
